unicode_smilies/unicode_smilies.php

```diff
@@ -1,260 +1,2042 @@
 <?php
 /*
  * Name: Unicode Smilies
- * Description: Smilies based on the unicode emojis - On Linux use https://github.com/eosrei/emojione-color-font to see them in color
+ * Description: Smilies based on the unicode emojis - On Linux use https://github.com/eosrei/emojione-color-font to see them in color and http://www.unicode.org/emoji/charts/full-emoji-list.html
  * Version: 1.0
  * Author: Michael Vogel <http://pirati.ca/profile/heluecht>
```

Comment:
I think your expansion is worth a higher version number, and maybe a mention in the Author list (just add another `* Author: ...` line if you like),

Reply:
okay